modules/special-rules/pages/cavalry.adoc

```diff
@@ -6,9 +6,10 @@ Of course, riders and their mounts vary enormously -- humans on horses, denizens
 
 ---
 
-Apply these rules to {cavalry} units, relative to the base units:
+Apply these rules to {cavalry} units:
 
-* Double their Speed value.
-* During the Assault phase, triple their Speed value if they charge into close combat.
-* For the purposes of movement, treat {cavalry} units as Vehicle units -- mounts find it difficult to take cover, to move through buildings or rubble, and so on.
-* Add +1 to their Armour value, up to a maximum of 6+.
\ No newline at end of file
+* Double their original Speed value.
+* During Assault moves, their original Speed value is tripled if this will bring them into base contact with enemy units, or doubled if it won't.
+* Treat {cavalry} units as Vehicle units -- mounts find it difficult to take cover, to move through buildings or rubble, and so on.
+
+NOTE: The part of {cavalry} that increased the unit's Armour value in 3rd edition has been split off into the new xref:armour-1.adoc[Armour +1] ability.
```

Comment:
1. It's not 'their original Speed value', at all. It's the Speed value of the unit they are based on in the army list. Hence "relative to their base units" in the lead-in sentence.
2. Phrases like "is doubled" are passive case.
3. Making base contact is a charge move by definition though the player should announce it as such — and I think I cleared that up a bit in my first pass at that those rules. So, we can and should express it as such I think.
4. The first bullet already says to double the Speed value (relative to the base unit). So there is no need to state this again for the Assault move, as it is in effect their new baseline movement value. We only need to mention that we triple the (base unit's) Speed value for a charge (in place of the double).
5. I don't think that we should imply that Cavalry units are effectively Vehicle units in all circumstances.

```suggestion
* Double their Speed value.
* Triple their Speed value instead, if they charge into close combat in the Assault phase.
* For movement, treat {cavalry} units as Vehicle units -- mounts find it difficult to take cover, to move through buildings or rubble, and so on.

NOTE: In comparison to _Epic 40,000_, we've moved the part of {cavalry} that increased the unit's Armour value to the new {armour-1} ability.
```
It's good to link to the new ability, but better as a related info link I think. Though I'm prepared to concede that if you think that there is some special reason to keep it in the note for this particular use case.

Reply:
Internal design notes:

I currently have Cavalry and Jump Packs categorised as unit abilities, **not** as unit design modifiers. Although the profile is altered, this categorisation was chosen because both abilities include in-game rules which alter how far the units can move into contact, and Jump Packs units being able to move over terrain.

In addition, both abilities have to take into account how we choose to display the 'full' unit profile. The detachment printer currently uses '[original unit Speed] (x2/x3*)' as I found it to be the least ambiguous option. So Assault Marines currently have Speed '10 cm (x2/x3*)' with '*Jump Packs' in the notes section.

And then there are the 60 cm Assault move Assault Marines that were recently discussed in one of the FB groups. The original GW text was very ambiguous - if the basic Speed on the profile is doubled, and they get to triple their Speed if they can contact the enemy, surely that means their Speed becomes 60cm? Your text is better, but it would need to be written as explicitly overriding the standard 'double the Speed shown on your profile' rule for reaching base contact.

So all of my edits to Cavalry and Jump Packs are informed by those decisions and factors.

Replying to the specific bullet points:

1. See above. When written as an in-game ability, there is no such thing as a 'base unit' so I removed all text referring to base units.
2. As you've worked out by now, active case versus passive case isn't one of my strengths. 😕
3.  This brings up a related problem. The current rules logic for Assault moves has been extensively discussed on FB and playtested (although they could do with more playtesting) and can be found at the end of the playtest file: https://www.facebook.com/groups/1952676958177757/permalink/3634224546689648. There is no separate concept of a charge move and the way that GW divided it up (and you inherited) has caused far more trouble than it's worth. Instead, the section for Assault moves will just need a bullet saying that Infantry can double their Speed if they can contact the enemy.
4. See the 60 cm Assault move Assault Marines. We can say that the actual Speed value has been doubled, but that will require a rewrite of the rest of the rule so that their Assault move Speed if contacting is an **exception** to the normal Assault move rules. Ideally written as 1.5x their listed Speed instead of the usual 2x listed Speed.
5. Counter-point - in what circumstances do you think that Cavalry **aren't** effectively Vehicles? Being transported? 'Cavalry [units] are counted as vehicles' is remarkably unambiguous for GW. We know they have to be Vehicles for movement because buildings are mentioned. We know they have to be Vehicles for shooting, because of the reference to not taking cover properly. 

In summary, we **could** rewrite Cavalry as a profile modifier instead of a unit ability and use '20 cm/30 cm*' or equivalent in the profile display, and make sure the Assault move distances are written as exceptions to the normal rules, but that would have knock-on effects for Jump Packs which would still include in-game rules. And yes, I know part of the problem is that I introduced categories for abilities, but I think that's useful enough way of categorising them that its more important than the problems it causes for Cavalry and Jump Packs.

In an ideal world, we could list split movement rates like '20 cm/30 cm' and the data sheet and Assault phase sections would explain what this means. Then Cavalry could become purely a profile modifier, and Jump Packs would be split into a profile modifier ability and a separate in-game ability. But I fear that would be another friction point for existing players.

Hmm... that's giving me ideas. I'll flesh them out a bit, and put them in a new post.

And finally... the link to Armour +1 is in the note because I don't regard it as related information in the context of Remastered. It's only related within the context of players coming from 3rd edition, and that a lot of units that have Cavalry will also have Armour + 1.

Reply:
### Cavalry and Jump Packs

**Assumption** - profile modifier abilities are not referenced during the game. They appear in the Detachment lists as shorthand, but apart from that they're primarily a unit design element with no in-game rules.


1. In Data sheets and characteristics > Speed, add:

> Where a unit has a split Speed value such as 20 cm/30cm, you use the first value for all purposes except Assault moves.

2. In the final Assault moves text, add something like this:

> Where a unit has a split Speed value such as 20 cm/30cm, you may use the second value if this would bring the unit into contact with an enemy. You **cannot** double the Speed value of the unit during an Assault move.

3. Cavalry becomes a pure profile modifier. It would still be listed in the notes section of the full expanded profile, but within brackets (or similar) to show that it's already been applied:

> Where your unit has the {cavalry} ability, apply these changes relative to their base unit:
> * The unit has a split Speed value. The first value is double the base unit's Speed value and the second is triple the base unit's Speed value.
> * The unit becomes a Vehicle unit.

4. Jump Packs has a stub page in the unit modifier list, call it Jump Packs (profile modification) or similar, which links to the main Jump Packs page. The stub page is there purely for consistency so that all profile modifiers are listed together.

5. The Jump Packs page has two sections (very rough draft):

> Where your unit has the {jump-packs} ability, apply this change relative to their base unit:
> * The unit has a split Speed value. The first value is double the base unit's Speed value and the second is triple the base unit's Speed value.
> 
> Where your unit has the {jump-packs} ability, apply these rules during the game:
> * While moving the unit, you can ignore the effects of terrain as long as the unit doesn't end their move in the terrain.
> * During Assault moves, you can move the unit over enemy units that have already been contacted.

_Editing note - I've tried to give the agency to the player in bullet 5, but it's awkward and the terrain and movement page currently gives the agency to the unit anyway._

Alternatively, for points 4 and 5, put the profile modification into Jump Packs (profile modification) and give both pages 'relevant information' links to each other.